### ISSUE TITLE
pipe fails to be initized

### ISSUE THREAD

**Issue by jjiikkkk** [Yuanshi9815/OminiControl#83]
section 1
%cd /content/OminiControl

import torch
from PIL import Image

from optimum.quanto import freeze, qfloat8, quantize
from diffusers import FlowMatchEulerDiscreteScheduler, AutoencoderKL
from diffusers.models.transformers.transformer_flux import FluxTransformer2DModel
from diffusers.pipelines.flux.pipeline_flux import FluxPipeline
from transformers import CLIPTextModel, CLIPTokenizer, T5EncoderModel, T5TokenizerFast

from src.flux.condition import Condition 
from src.flux. generate import generate

with torch.inference_mode():
    dtype = torch.bfloat16
    model = "/content/model"
    scheduler = FlowMatchEulerDiscreteScheduler.from_pretrained(model, subfolder="scheduler")
    tokenizer = CLIPTokenizer.from_pretrained(model, subfolder="tokenizer", torch_dtype=dtype)
    tokenizer_2 = T5TokenizerFast.from_pretrained(model, subfolder="tokenizer_2", torch_dtype=dtype)

    text_encoder = torch.load('/content/8bit/text_encoder.pt')
    text_encoder.eval()
    text_encoder_2 = T5EncoderModel.from_pretrained("/content/clip", torch_dtype=dtype)
    text_encoder_2.eval()
    transformer = torch.load('/content/8bit/transformer.pt')
    transformer.eval()
    vae = torch.load('/content/8bit/vae.pt')
    vae.eval()

    pipe = FluxPipeline(scheduler=scheduler, tokenizer=tokenizer, tokenizer_2=tokenizer_2, text_encoder=None, text_encoder_2=None, transformer=None, vae=None)
    pipe.text_encoder = text_encoder
    pipe.text_encoder_2 = text_encoder_2
    pipe.transformer = transformer
    pipe.vae = vae
    pipe.load_lora_weights("/content/lora", weight_name=f"subject_512.safetensors", adapter_name="subject",)
    pipe.to('cuda')
     

section 2

import os
import sys
from PIL import Image
import torch

# 将目录添加到 sys.path
sys.path.append('/content/OminiControl/src')

# 导入模块
from flux.condition import Condition
from flux.generate import generate

with torch.inference_mode():
    input_image = "/content/test.webp"
    prompt = "A very close up view of this item. It is placed on a wooden table. The background is a dark room, the TV is on, and the screen is showing a cooking show. With text on the screen that reads 'Omini Control!'"
    image = Image.open(input_image)
    w, h, min_size = image.size[0], image.size[1], min(image.size)
    image = image.crop(((w - min_size) // 2, (h - min_size) // 2, (w + min_size) // 2, (h + min_size) // 2,))
    image = image.resize((512, 512))
    condition = Condition("subject", image)
    output_image = generate(pipe, prompt=prompt.strip(), conditions=[condition], num_inference_steps=8, height=512, width=512).images[0]
output_image  

  here, always unable to find out  pipe

**Comment by jjiikkkk** [comment]
image = Image.open("assets/penguin.jpg").convert("RGB").resize((512, 512))

condition = Condition("subject", image, position_delta=(0, 32))

prompt = "On Christmas evening, on a crowded sidewalk, this item sits on the road, covered in snow and wearing a Christmas hat."


seed_everything(0)

result_img = generate(
    pipe,
    prompt=prompt,
    conditions=[condition],
    num_inference_steps=8,
    height=512,
    width=512,
).images[0]

concat_image = Image.new("RGB", (1024, 512))
concat_image.paste(image, (0, 0))
concat_image.paste(result_img, (512, 0))
concat_image

this section, also unable to find out pipe
